### ISSUE TITLE
Showing red "error" highlighting on random JS files

### ISSUE THREAD

**Issue by GeorgeWL** [jest-community/vscode-jest#908]
Type: <b>Bug</b>

Seems to be somewhat inconsistent, only was able to find it was this extension due to running the VS Code "Bisect" tool

Sometimes, when open a js file (not a spec.js, just a normal js file), random lines of code get highlighted in red, the colour usually reserved for "there's an error in this line of code"

It seems to only happen after a test suite has run, and had one or more failures.

I've found that when disable the extension, never see this bug.

Extension version: 4.6.0
VS Code version: Code 1.71.0 (784b0177c56c607789f9638da7b6bf3230d47a8c, 2022-09-01T07:36:10.600Z)
OS version: Windows_NT x64 10.0.19044
Modes:
Sandboxed: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-10850H CPU @ 2.70GHz (12 x 2712)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: disabled_off|
|Load (avg)|undefined|
|Memory (System)|31.76GB (16.53GB free)|
|Process Argv|. --crash-reporter-id efc46c73-0787-4ddc-bbea-c3adbc4b7bc4|
|Screen Reader|no|
|VM|0%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383cf:30185419
vspor879:30202332
vspor708:30202333
vspor363:30204092
vslsvsres303:30308271
pythonvspyl392:30443607
vserr242cf:30382550
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vshan820:30294714
vstes263:30335439
vscoreces:30445986
pythondataviewer:30285071
vscod805:30301674
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
cmake_vspar411cf:30557515
vsaa593cf:30376535
pythonvs932:30410667
cppdebug:30492333
vsclangdf:30486550
c4g48928:30535728
hb751961:30553087
dsvsc012cf:30540253
azure-dev_surveyone:30548225
i497e931:30553904
fc301958:30567733

```

</details>

**Comment by connectdotz** [comment]
hmm... the extension did mark failed test with red underlying (is that what you saw?). But it should never mark non-test files, as long as the jest config and source-map are correct... did you see any error in the console log? or if you can provide a sample repo, we can take a closer look.

**Comment by GeorgeWL** [comment]
I can't provide a sample repo, but I'll open the logs next I see it, it seems to be rather intermittent.

**Comment by mmarzullotv** [comment]
Coming from VS Code and seems like many are having the same issue due to this extension. After bisecting turned out to be this one. I will disable it for now until fixed.

**Comment by connectdotz** [comment]
@mmarzullotv thanks for bringing this to our attention. This one has opened 4 months ago, and we could not reproduce it. You are the 2nd person reported this, so would be really great if you can share a reproducible use-case to help us diagnose further. 

We fixed an issue in [v5.2.0 pre-release](https://github.com/jest-community/vscode-jest/blob/master/release-notes/release-note-v5.x.md#v520-pre-release) that could be related. Do you mind give it a try?

If you still see problems, can you show a screencast? Is there any pattern? Is this a multi-root? What is your workspace set-up and customization?

**Comment by GeorgeWL** [comment]
Can confirm I've not seen it since that update
